pom.xml

```diff
@@ -214,6 +214,7 @@
       <plugin>
         <groupId>com.github.github</groupId>
         <artifactId>site-maven-plugin</artifactId>
+        <version>0.12</version>
```

Comment:
This should be unnecessary, since it pulls the version info from the parent pom. Simply updating fcrepo4 will have the same effect.